docs/content/guides/getting-started/introduction.md

```diff
@@ -41,15 +41,15 @@ Thousands of business apps depend on Handsontable for entering, editing, validat
 ::: only-for javascript
 To jump straight into the sample code, open the demo app at CodeSandbox:
 
-- [JavaScript demo](https://codesandbox.io/s/handsontable-javascript-data-grid-hello-world-app-forked-33g8wu)
-- [React demo](https://codesandbox.io/s/handsontable-react-data-grid-hello-world-app-forked-j2vdws)
-- [Angular demo](https://codesandbox.io/s/handsontable-angular-data-grid-hello-world-app-forked-st2uyz)
-- [Vue 2 demo](https://codesandbox.io/s/handsontable-vue-data-grid-hello-world-app-forked-z288wy)
-- [TypeScript demo](https://codesandbox.io/s/handsontable-typescript-data-grid-hello-world-app-forked-7xzs5w)
+- [JavaScript demo](https://codesandbox.io/s/handsontable-javascript-data-grid-hello-world-app-12-3-2-updhi7)
+- [React demo](https://codesandbox.io/s/handsontable-react-data-grid-hello-world-app-12-3-2-hqy9xi)
```

Comment:
I can see that you bumped the React version in these to `18`. This will result in this warning being thrown:
```
Warning: ReactDOM.render is no longer supported in React 18. 
```

Bumping the React version would require additional changes to the demos: https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis

Reply:
@jansiegel – ok, going back to `16.4.0`: https://codesandbox.io/s/handsontable-react-data-grid-hello-world-app-12-3-2-hqy9xi?file=/package.json